Source/Eto.Gtk/Forms/Controls/SplitterHandler.cs

```diff
@@ -114,6 +114,38 @@ void Create()
 			}
 			if (position != null)
 				Control.Position = position.Value;
+
+			if (fixedPanel != SplitterFixedPanel.Panel1)
+				Control.SizeAllocated += Control_SizeAllocated;
+		}
+
+		void Control_SizeAllocated(object o, Gtk.SizeAllocatedArgs args)
+		{
+			Control.SizeAllocated -= Control_SizeAllocated;
```

Comment:
...but this may be a problem, I am not sure how would that behave when removed from parent and then added again or to different widget